src/table/column-filter.scss

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@import '../style/index';
+@import '../menu/index';
+
+@mixin iui-column-filter {
+  @include iui-dropdown;
+  padding: $iui-baseline $iui-sm;
+  width: fit-content;
```

Comment:
`width: fit-content` is not supported by IE. You can achieve the same with:
![image](https://user-images.githubusercontent.com/36186912/116520833-ea787880-a8db-11eb-9bdc-5b0da761a42d.png)
(I don't know why width is crossed but it is needed.)

Reply:
If I remove `width: fit-content;` I get uneven text fields.
![Screen Shot 2021-04-29 at 11 06 18 AM](https://user-images.githubusercontent.com/849817/116573584-cad45680-a8c1-11eb-86ec-b99d6f75af7d.png)

So then I tried applying `width: 100%;` to `.iui-input-container` and that messes up the other filter options.
![Screen Shot 2021-04-29 at 11 07 26 AM](https://user-images.githubusercontent.com/849817/116573762-f22b2380-a8c1-11eb-8163-a4ea5de8394e.png)

Unsure how to fix this at the moment. 🤔

Reply:
Figured it out! I can apply `width: 100%` to `.iui-input-container:not(.iui-inline)`.